.travis.yml

```diff
@@ -17,19 +25,21 @@ env:
 matrix:
   include:
     - php: '7.1'
-      env: DB=MYSQL RECIPE_VERSION=4.2.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: '7.1'
-      env: DB=PGSQL RECIPE_VERSION=4.3.x-dev PHPUNIT_COVERAGE_TEST=1
+      env: DB=PGSQL PHPUNIT_TEST=1
     - php: '7.2'
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev BEHAT_TEST=1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev NPM_TEST=1
+      env: DB=MYSQL BEHAT_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
```

Comment:
Why have you removed these versions? This removes our ability to test against multiple versions of SilverStripe, which is a very important part of these build matrixes